src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/abstractItems/AContainer.java

```diff
@@ -238,31 +240,30 @@ protected void tick(Block b) {
         BlockMenu inv = BlockStorage.getInventory(b);
 
         if (isProcessing(b)) {
-            int timeleft = progress.get(b);
 
-            if (timeleft > 0) {
-                ChestMenuUtils.updateProgressbar(inv, 22, timeleft, processing.get(b).getTicks(), getProgressBar());
+            if (takeCharge(b.getLocation())) {
```

Comment:
Why was this now placed in front of everything?

Reply:
because it should be called in both cases, so might as well right?

Reply:
huh, you are right.